.github/workflows/check-e2e.yml

```diff
@@ -58,6 +61,18 @@ jobs:
         if: steps.playwright-dep-cache.outputs.cache-hit == 'true'
         run: npx playwright install-deps
 
+      - id: auth
+        uses: google-github-actions/auth@v1
+        with:
+          workload_identity_provider: 'projects/281042598092/locations/global/workloadIdentityPools/github-actions/providers/google-cloud'
+          service_account: 'github-actions@radicle-services.iam.gserviceaccount.com'
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Download radicle-httpd executable
+        run: gsutil cp gs://heartwood-artifacts/119870d59683c446b2377974ab31bc1fe1b98d43/radicle-httpd ./
```

Comment:
We could put the binary in `tests/tmp`. And then run the httpd below with:
```
PATH=./tests/tmp:$PATH ./scripts/run-httpd-with-fixtures --non-interactive --detach --binary
```

Reply:
✅